apps/cli/src/commands/profile/list.profile.ts

```diff
@@ -4,7 +4,10 @@ import {
 } from 'src/types/command/command.types'
 import BaseCommand from '../base.command'
 import { fetchProfileConfig } from '../../util/configuration'
-import Logger from '../../util/logger'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Table = require('cli-table')
```

Comment:
We don't want to use the `require` syntax.

Reply:
It is not working without require for some reason